WordPress/Classes/ViewRelated/Notifications/Views/NoteTableViewCell.swift

```diff
@@ -79,41 +79,35 @@ import Foundation
     public class func reuseIdentifier() -> String {
         return classNameWithoutNamespaces()
     }
-    
+
+    public func downloadIconWithURL(url: NSURL?) {
+        let isGravatarURL = url.map { Gravatar.isGravatarURL($0) } ?? true
```

Comment:
The second branch of the _if_ isn't entering when the Gravatar URL is the unknown email. (So that we loose the custom tinted placeholder)